dune

```diff
@@ -1 +1 @@
-(ignored_subdirs (node_modules js))
+(data_only_dirs node_modules js)
```

Comment:
this should be `(dirs :standard \ node_modules js)`.

`data_only_dirs` is a different concept.